hvplot/converter.py

```diff
@@ -2631,14 +2631,15 @@ def ohlc(self, x=None, y=None, data=None):
         tools = seg_cur_opts.pop('tools', [])
         if 'hover' in tools:
             tools[tools.index('hover')] = HoverTool(
+                formatters={f'@{x}': 'datetime'},
```

Comment:
You still need support the case where x is not datetime.

Reply:
Done